core/src/main/java/io/grpc/internal/ServerImpl.java

```diff
@@ -133,6 +133,7 @@
     this.fallbackRegistry =
         Preconditions.checkNotNull(builder.fallbackRegistry, "fallbackRegistry");
     this.transportServer = Preconditions.checkNotNull(transportServer, "transportServer");
+    this.logId = InternalLogId.allocate("Server", /*details=*/ null);
```

Comment:
Also include `transportServer.getPort()`?

Reply:
I was a little worried that when we move to listening on multiple ports, this would be wrong.    I can add it, but I think it may be removed soon.